src/main/java/quotify_app/entities/PredictionRequestBuilder.java

```diff
@@ -0,0 +1,206 @@
+package quotify_app.entities;
+
+/**
+ * A builder for constructing instances of {@link PredictionRequest}.
+ */
+public final class PredictionRequestBuilder {
+    private int beds;
+    private int baths;
+    private double universalsize;
```

Comment:
same convention as before this will be fixed when u refactor with intellij though.

Reply:
Oh, I left these like this because the http request expects the fields to match these names exactly. If they aren't passing checkstyle, I'll figure out a way to change them, but if its not complaining, I'd suggest we leave these be, since when we construct the http request, it involves converting an object into a JSON string, and the object has fields given by the names of these class variables